doc/nmstatectl.8.in

```diff
@@ -7,10 +7,16 @@ nmstatectl \- A nmstate command line tool
 .br
 .B nmstatectl show [\fB-r, --running-config\fR]
 .br
+.B nmstatectl s \fR[\fIINTERFACE_NAME\fR] [\fB--json\fR]
```

Comment:
This give user impression that `nmstatectl s` is a new subcommand, and ever time we have new option added to `nmtatectl show`, we have to do it for this `nmstatectl s`.

I prefer not listing it at top level, just mention in description of `show` that you can use `s` as alias to save typing.

Reply:
> This give user impression that `nmstatectl s` is a new subcommand, and every time we have new option added to `nmtatectl show`, we have to do it for this `nmstatectl s`.
> 
> I prefer not listing it at top level, just mention in description of `show` that you can use `s` as alias to save typing.

thanks. updated, but ci is not happy yet

Reply:
happy now.
can you review please